src/BotPlutusInterface/CardanoCLI.hs

```diff
@@ -480,8 +481,8 @@ networkOpt pabConf = case pabConf.pcNetwork of
   Mainnet -> ["--mainnet"]
 
 txOutRefToCliArg :: TxOutRef -> Text
-txOutRefToCliArg (TxOutRef (TxId txId) txIx) =
-  encodeByteString (fromBuiltin txId) <> "#" <> showText txIx
+txOutRefToCliArg (TxOutRef (TxId tId) txIx) =
```

Comment:
[nits] I think there's a typo here

Reply:
Had to rename that because we are importing txId from Ledger.Tx